Makefile

```diff
@@ -107,7 +111,13 @@ package-deb: build
 
 .PHONY: docker-build
 docker-build: amd64-build
-	docker build -t $(DOCKER_NAMESPACE)/$(COMPONENT):$(VERSION) -f ./cmd/$(COMPONENT)/Dockerfile .
+	chmod +x cmd/awscollector/build_docker_binary.sh
+	docker buildx build --platform linux/amd64 -t $(DOCKER_NAMESPACE)/$(COMPONENT):$(VERSION) -f ./cmd/$(COMPONENT)/Dockerfile .
+
+.PHONY: docker-build-arm
+docker-build-arm: arm64-build
+	chmod +x cmd/awscollector/build_docker_binary.sh
```

Comment:
Why not just include the chmod step inside the Dockerfile if it's required for each run?

Reply:
done